packages/synapse-react-client/src/components/OAuthClientManagement/CreateOAuthClient.tsx

```diff
@@ -98,13 +110,17 @@ export const CreateOAuthModal: React.FunctionComponent<
     )
     setPolicyUri(client?.policy_uri ?? '')
     setClientUri(client?.client_uri ?? '')
-    setSectorUri(client?.sector_identifier_uri ?? undefined)
+    setSectorUri(client?.sector_identifier_uri ?? '')
+    setUserinfoSignedResponseAlg(
+      client?.userinfo_signed_response_alg ??
+        defaultUserInfoSignedResponseAlgorithm,
+    )
     setTosUri(client?.tos_uri ?? '')
   }, [isShowingModal, client])
 
   useDebouncedEffect(
     () => {
-      if (accessToken) {
+      if (accessToken && oAuthClient.client_id) {
```

Comment:
Prevent a 400 when calling `isOAuthClientReverificationRequired` when `client_id` is undefined